### DISCUSSION TITLE
How access local LLM without Ollama?

### DISCUSSION THREAD

**Discussion by mobguang** [vanna-ai/vanna#430] [Q&A]
**Describe the bug**
I cannot find documentation for explain how to access local LLM without Ollama. 

**To Reproduce**
I have downloaded CodeLlama-7b-Instruct-hf on my Mac, but there is no example on Vanna.ai documentation to describe how to use this kind of LLM.

**Expected behavior**
May I know is there any way to access this kind of LLM? Thanks in advance.

**Error logs/Screenshots**
If applicable, add logs/screenshots to give more information about the issue.

**Desktop (please complete the following information where):**
 - OS: Macos
 - Version:  14.4.1
 - Python: 3.11
 - Vanna: 0.5.4

**Additional context**

**Comment by zainhoda** [comment]
You'd have to follow the documentation for "Other LLM"

https://vanna.ai/docs/postgres-other-llm-vannadb/

**Reply by mobguang** [reply]
@zainhoda 

Thanks for your reply. 

As I'm a beginner of Vanna.ai, could you please provide more detail about the how to use "Other LLM"? The doc from https://vanna.ai/docs/postgres-other-llm-vannadb/ is not thorough enough.
